MyOwnList.Test/AddTests/AddEndTest.cs

```diff
@@ -5,18 +5,20 @@ namespace MyOwnList.Test.AddTests
 {
     class AddEndTest
```

Comment:
Bad approach. It could be good when you have at least 3-5 parametrized really big tests. This tests are small and all of them are linked with one class. Please move it to one file